# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/shared/PFileSystemManager.ipdl
# Commit: 302fdb94bf7f
# Full Hash: 302fdb94bf7f5cd817b69fc4eaaaf6ff8b62bec8
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2023-02-06 21:36:23
# Regressor Bug: 1809064
# File Overlap Count: 1
# Description:
#   Bug 1809064 - Use managed endpoints for PFileSystemAccessHandle construction; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166342
# ==============================================================================

diff -r 100b45bcf622 -r 302fdb94bf7f dom/fs/shared/PFileSystemManager.ipdl
--- a/dom/fs/shared/PFileSystemManager.ipdl	Mon Feb 06 15:01:20 2023 +0000
+++ b/dom/fs/shared/PFileSystemManager.ipdl	Mon Feb 06 15:10:34 2023 +0000
@@ -131,7 +131,7 @@
 struct FileSystemAccessHandleProperties
 {
   RandomAccessStreamParams streamParams;
-  PFileSystemAccessHandle accessHandle;
+  ManagedEndpoint<PFileSystemAccessHandleChild> accessHandleChildEndpoint;
 };
 
 union FileSystemGetAccessHandleResponse
@@ -413,7 +413,6 @@
       returns (uint64_t aQuotaGranted);
 
  child:
-  async PFileSystemAccessHandle();
   async PFileSystemWritableFileStream();
 
   async CloseAll()
